### PR TITLE
Remove all inter-feature dependencies for 'meeting'

### PR DESCRIPTION
**Pull request by Tyratox**

### CODE REVIEW

**Review by 0xdieman — APPROVED** [review]
I feel like my reviews are not very useful when I do not find anything to say but it seems all good to me 😂

File: fe1-web/src/features/meeting/interface/Feature.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  }
+
+  export interface Event {
+    id: Hash;
+    name: string;
+  }
+
+  /* export enum EventType {
+    MEETING = 'MEETING',
+  } */
```
[the remark this answers is not in the document]
Comment (Tyratox) [reply fe1-web/src/features/meeting/interface/Feature.ts:29]: Oops, thank you!

**Comment by Tyratox** [comment]
Yes I am on a PR for splitting the event reducer :) I wanted to do some preparations with this and the roll call PR 😄

**Review by pierluca — APPROVED** [review]
LGTM

**Pull request merged**